### ISSUE TITLE
 weather, location functions doesnt work

### ISSUE THREAD

**Issue by amirfahoum** [WebAhead5/FaceDetect#10]
great idea , every one can use it to know the location and weather status. 
but there are some problems i just face ...
the web doesn't display  weather and location data...

**Comment by MoradAbed** [comment]
Yeah, it's an issue related to the variety of versions of the browsers. 
Working on solving this Ameer, thanks!
